src/app/apiTest/hooks/todoQueries.ts

```diff
@@ -0,0 +1,9 @@
+import { useQuery, UseQueryResult } from '@tanstack/react-query';
+import { getAllByUserId, getOneByTodoId } from '@/app/apiTest/calls/todoCalls';
+import { SelectTodo } from '@/db/schema/todos';
+
+export const useGetAll = (userId: string): UseQueryResult<SelectTodo[], Error> =>
+  useQuery({ queryKey: ['todos', userId], queryFn: () => getAllByUserId(userId), enabled: true });
```

Comment:
```suggestion
export const useGetAll = (userId: string): UseQueryResult<SelectTodo[], Error> =>
  useQuery({ queryKey: ['todos', userId], queryFn: () => getAllByUserId(userId), enabled: !!userId });
```
위 api는 userId 가 없으면 호출되면 안되는 것일테니 enabled속성에 userId가 있는지 없는지로 넣어주는 것이 더 맞을 것 같은데 어떤가요?

Reply:
감사합니다 수정할게요 :)